internal/command/meta_backend.go

```diff
@@ -1021,7 +1021,9 @@ func (m *Meta) backend_C_r_S_changed(c *configs.Backend, cHash int, sMgr *clista
 		if c.Type == "cloud" {
 			output = fmt.Sprintf(outputBackendMigrateChangeCloud, s.Backend.Type)
 		}
-		m.Ui.Output(strings.TrimSpace(output))
+		m.Ui.Output(m.Colorize().Color(fmt.Sprintf(
```

Comment:
Noting that we should try to avoid calling `Color()` on the output of `fmt.Sprintf`, and rather call it on the unprocessed format string. This avoids accidentally interpreting user input as color commands.

That might be awkward to fix here, and we know that the interpolated values will not include color commands, so I'm ✅ for now. Just a heads up for the future.